src/ontology/agro-edit.owl

```diff
@@ -5446,6 +5462,127 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010155> "i
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010156> "2021-12-24T00:21:23Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/AGRO_00010156> "fenceline planting")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010156> "live fence"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010156> <http://purl.obolibrary.org/obo/ENVO_01000468>)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010156> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000056> <http://purl.obolibrary.org/obo/AGRO_00000584>))
+
+# Class: <http://purl.obolibrary.org/obo/AGRO_00010157> (boundary planting)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "Technical Specification Boundary Planting. https://takingroot.org/wp-content/uploads/2015/04/takingroot-boundary-planting.pdf") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AGRO_00010157> "A planting process where trees or hedges are planted along boundaries to delimitate property, agricultural fields, pastures, roads or any other place where fences might be located."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/AGRO_00010157> "https://orcid.org/0000-0003-3985-4911")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010157> "2021-12-27T16:54:13Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010157> "boundary planting"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010157> <http://purl.obolibrary.org/obo/AGRO_00000231>)
+
+# Class: <http://purl.obolibrary.org/obo/AGRO_00010158> (windbreak)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://www.fs.usda.gov/nac/practices/windbreaks.php") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AGRO_00010158> "Windbreaks are linear plantings of trees and shrubs designed to enhance crop production, protect people and livestock, and benefit soil and water conservation."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/AGRO_00010158> <https://orcid.org/0000-0003-3985-4911>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010158> "2022-01-03T23:59:15Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/AGRO_00010158> "shelterbelt")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010158> "windbreak"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010158> <http://purl.obolibrary.org/obo/AGRO_00000584>)
```

Comment:
I wonder if the subcass: http://purl.obolibrary.org/obo/ENVO_01000408 would be better? Again, this is not a process but the result of the process of planting trees.